### DISCUSSION TITLE
How to align content using Markdown Wysiwyg editor ?

### DISCUSSION THREAD

**Discussion by Skwall** [OrchardCMS/OrchardCore#12011] [Q&A] [answered]
Hello !
I'm using orchard core to create a blog, and I'm wondering how to set a particular alignment to content such as paragraphs or images using the markdown editor when creating a blog post ? (I'm using the Wysiwyg editor)
I did not found this option.

Thanks :)

**Comment by hishamco** [comment]
Did you check the editor settiings

**Reply by hishamco** [reply]
Wait a minute, mardown doesn't support alignment except in tables AFAIK. BTW there are a varity of settings that you can configure the `MarkdownBody` please check https://docs.orchardcore.net/en/latest/docs/reference/modules/Markdown/#markdown-configuration

**Comment by ns8482e** [comment, marked as answer]
May be you can write custom short code and use it in markdown.

**Update**
Here's sample sort code you can create.

Usage
`[align text=justify]this is justify text[/align]`

Code 
```liquid
<div style="text-align:{{Args.Text}}">{{Model.Content}}</div>
```

**Reply by hishamco** [reply]
But how it will change the text alignment?

**Reply by hishamco** [reply]
Seems you need to write shortcode as @ns8482e mention, that surround a text into a paragraph, then set the `align` property

**Reply by Skrypt** [reply]
There is no such thing as "alignment" with Markdown generally.
The only way to align is to use a Table.

```markdown
| Default Header | Left Align | Right Align | Center Align |
| -------------- | :--------- | ----------: | :----------: |
```

Anything else would be shortcode like mentioned.

**Reply by hishamco** [reply]
> There is no such thing as "alignment" with Markdown generally.
> The only way to align is to use a Table.

Already mention [here](https://github.com/OrchardCMS/OrchardCore/discussions/12011#discussioncomment-3148106) ;)

**Reply by hishamco** [reply]
> There is no such thing as "alignment" with Markdown generally.
> The only way to align is to use a Table.

Already mention [here](https://github.com/OrchardCMS/OrchardCore/discussions/12011#discussioncomment-3148106) ;)

**Comment by Skwall** [comment]
I'll create a shortcode then, thank you guys for all your answers !
